Makefile

```diff
@@ -117,9 +117,9 @@ check:
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
-test-kind: build local-env-up ## Run integration tests against kind cluster
+test-kind: build #local-env-up ## Run integration tests against kind cluster
```

Comment:
This was committed by mistake wasn't it?

Reply:
Yep